src/views/layouts/playerManagement.hbs

```diff
@@ -173,7 +174,7 @@ function enableEdit(button) {
 }
 
 function resetTotalGoals() {
-    if (confirm('Sind Sie sicher, dass Sie alle totalen Tore zurücksetzen möchten?')) {
+    if (confirm('Sind Sie sicher, dass Sie alle Tore der Spieler zurücksetzen möchten? (löscht nur Tore aus Spielern nicht aus "Game")')) {
```

Comment:
this should also delete the assigned goals in the "torschützen" tab, which can then be newly assigned to players

Reply:
Ok i changed it